windows/src/engine/keyman32/addins.cpp

```diff
@@ -74,17 +74,9 @@ void ReadAddins(HKEY hkey)
 			_td->Addins[n].Uninitialise = NULL;
 			_td->Addins[n].ShouldProcess = NULL;
 			strcpy(_td->Addins[n].ClassName, buf);
-			reg->ReadString(buf, buf2, 512);
-			//char *p = strtok(buf2, ",");
-			//if(p)
-			//{
-				_td->Addins[n].Application[0] = 0;
-				//strcpy(Addins[n].Application, p);
-				//p = strtok(NULL, ",");
-				strcpy(_td->Addins[n].AddinName, buf2); //p);
-				//SendDebugMessageFormat(GetFocus(), sdmGlobal, 0, "Addins: ReadAddins: App=%s Addin=%s %d", Addins[n].Application, Addins[n].AddinName, Addins[n].hAddin);
-				n++;
-			//}
+			reg->ReadString(buf, _td->Addins[n].AddinName, 260);
```

Comment:
Where is `260` from?

Reply:
globals.h defines it as:

```
char AddinName[260];
```

I'm not trying to fix constants, etc, in this PR, just coverity reported issues.